react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecTextShadowNode.java

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.mobile.ReactNativeAztec;
+
+import android.support.annotation.Nullable;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.uimanager.annotations.ReactProp;
+import com.facebook.react.views.textinput.ReactTextInputShadowNode;
+
+public class ReactAztecTextShadowNode extends ReactTextInputShadowNode {
+    private @Nullable
```

Comment:
Def not a blocker, just noticed that this got bumped to two lines for some reason. Your earlier work had it on one: https://github.com/hypest/RN060-aztec-experiments/blob/5bc23a59dfadc47b97ac5381f33c14829bda280e/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecTextShadowNode.java#L10

Reply:
Ah, let me put them in a single line 👍

Reply:
Done with c57d61b.